docs/source/comparisons.rst

```diff
@@ -16,6 +16,13 @@ While HindcastEnsemble skill is computed over all initializations ``init`` of th
 PerfectModelEnsemble skill is computed over a supervector comprised of all initializations and members, which allows the computation of the ACC-based skill [Bushuk2018]_, but also returns a mean forecast skill over all initializations.
 The supervector approach shown in [Bushuk2018]_ and just calculating a distance-based metric like ``rmse`` over the member dimension as in [Griffies1997]_ yield very similar results.
 
+
+Compute over dimension
+######################
+
+The argument ``dim`` defines over which dimension a metric is computed. We can apply a metric over ``dims`` from [``'init'``, ``'member'``, ``['member', 'init']``] in :py:func:`climpred.prediction.compute_perfect_model` and [``'init'``, ``'member'``] in :py:func:`climpred.prediction.compute_hindcast`. The resulting skill is then reduced by this ``dim``. Therefore, applying a metric over dim='member' creates a skill for all initialisations individually. This can show the initial conditions dependence of skill. Likewise when computing skill over ``'init'``, we get skill for each member. This ``dim`` argument is different from the ``comparison`` argument which just specifies how ``forecast`` and ``reference`` are defined.
```

Comment:
Maybe say "the optional argument ...". Also check consistency of the keyword, you say "We can apply a metric over `dims`" should be `dim`?

Reply:
Do we need a `climpred.utils` function to check that `dim=XXX` is from your lists for compute hindcast and compute perfect model?  Could you convert `comparisons.rst` to a jupyter notebook and throw in a quick example with plots for this here?

Reply:
> Do we need a climpred.utils function to check that dim=XXX is from your lists for compute hindcast and compute perfect model?

I will include that.

> Could you convert comparisons.rst to a jupyter notebook and throw in a quick example with plots for this here?

there is only little difference in numerical values. sure, to see the different outputs. Its a more general thing: do we want to have so many notebooks? I think this dims argument is just a small detail

Reply:
> do we want to have so many notebooks? I think this dims argument is just a small detail

To clarify, I meant just switching it to a notebook so you could insert an example inline, and then use markdown cells for the rest of the details.

But in general I think the more extensive the documentation, the better. We want `climpred` to be as accessible as possible. So if there's a feature in it that can be used, it should be documented. But in this case I agree the text is fine for now.